packages/common/src/authorization/scopes.ts

```diff
@@ -514,6 +514,20 @@ const scopes: Scope[] = [
         group: ScopeGroup.DATA,
         getConditions: addDefaultOrgIdCondition,
     },
+    {
+        name: 'create:VirtualView',
+        description: 'Create virtual views',
+        isEnterprise: false,
+        group: ScopeGroup.DATA,
+        getConditions: addDefaultOrgIdCondition,
+    },
+    {
+        name: 'delete:VirtualView',
+        description: 'Delete virtual views',
+        isEnterprise: false,
+        group: ScopeGroup.DATA,
+        getConditions: addDefaultOrgIdCondition,
```

Comment:
TODO allow deleting your own created scope, can be done  on a separate PR

Reply:
I just added a modifier to scopes for this use-case to power Spaces a bit better. What do you think of defining those like `delete:VirtualView@self` or `manage:Space@self`. 

https://github.com/lightdash/lightdash/pull/16456/files#diff-1776342f7cd2898b6a4b47331c26e9fdb385fc3b180f8ac72e52aeb925ebd28eR114-R132

Reply:
I think it would be nice if we can assume for now that `create = create + delete self` and  `delete = delete from others`  and `manage = create + delete` 

Not sure if this is possible though 

I'm trying to prioritize the `user/dev` experience, over the permisison internal logic